pkg_specs/bitcoin-@variant.sps

```diff
@@ -89,6 +89,12 @@ try_overwrite_default = "mem_avail=\"`grep MemTotal /proc/meminfo | awk '{{print
 priority = "medium"
 summary = "Size of database cache in MB"
 
+[config."bitcoin.conf".ivars.nosettings]
+type = "bool"
+default = "true"
+priority = "medium"
+summary = "Disable read-write settings storage through GUI or RPC interfaces"
+
```

Comment:
I was playing with this before and this setting has a weird quirk: bitcoind bases its truthness on its presence/absence. So this would turn it on even if set to `false`!

This is what I came up with as a solution:

```
[config."bitcoin.conf".ivars.no_settings]
type = "bool"
default = "true"
priority = "low"
summary = "Disable dynamic settings"
store = false

[config."bitcoin.conf".hvars.nosettings]
type = "string"
script = "test \"${{CONFIG[\"bitcoin-{variant}/no_settings\"]}}\" = true && echo -n 1"
ignore_empty = true
```

If you have a better idea, I'd love to hear!

Reply:
@Kixunil Oh, I ignored that bitcoind bases its truthness on its presence/absence. For this, my idea is that there's no need for setting up a non-intuitive `no_settings` ivar, just leave some comment for easy modification.

```conf
[config."bitcoin.conf".hvars.nosettings]
type = "string"
# Change "true" into "false" in the following line if you don't want this.
script = "true && echo -n 1"
ignore_empty = true
summary = "Disable dynamic settings"
```

Will commit my idea into this PR first, if you don't like my idea, please don't hesitate to tell me.

Reply:
Oh, I completely missed that problem, good catch!

We absolutely can not hard-code such settings into specification files. We could force all people to run with `nosettings` but that doesn't seem great. Do you have some reason to not just modify my script?

[config."bitcoin.conf".ivars.no_settings]
type = "bool"
default = "true"
priority = "low"
summary = "Disable dynamic settings"
store = false

[config."bitcoin.conf".hvars.nosettings]
type = "string"
script = "if [ \"${{CONFIG[\"bitcoin-{variant}/no_settings\"]}}\" = true ]; then echo -n 1; fi"
ignore_empty = true

Reply:
> We absolutely can not hard-code such settings into specification files. We could force all people to run with `nosettings` but that doesn't seem great. 

OK, I see. In that case I won't come up with a better solution, so will adopt your idea. Thanks!

Reply:
@Kixunil  Updated!